charts/k8ssandra/values.yaml

```diff
@@ -523,9 +523,9 @@ medusa:
   # https://github.com/thelastpickle/cassandra-medusa/tree/master/k8s.
   image:
     # -- Specifies the container repository for Medusa
-    repository: docker.io/k8ssandra/medusa
+    repository: localhost:5000/medusa
```

Comment:
This is temporary, waiting for https://github.com/thelastpickle/cassandra-medusa/pull/366.